### ISSUE TITLE
Can't access my project, always get 'Congratulations page'

### ISSUE THREAD

**Issue by alenn-m** [puphpet/puphpet#2209]
I'm using Ubuntu 14.04 as working machine. When I visit local.puphpet I get the same page which I see when I visit the 192.168.56.101 (Congratulations page).

This is yaml file: http://pastebin.com/XURSUD2f
This is /etc/hosts file:

127.0.0.1 localhost
127.0.1.1 alen-Aspire-E1-570G
192.168.10.10 homestead.app
192.168.56.101 local.puphpet

I'm using latest version of VirtualBox.

**Comment by alenn-m** [comment]
Update:
I installed vagrant-hostmanager extension, that extension added this to my hosts file:
## vagrant-hostmanager-start id: 9b113ff3-4409-4d22-b7f3-7015c52a1250
192.168.56.101	local.puphpet
192.168.56.101	snoop.dev
192.168.56.101	www.snoop.dev
## vagrant-hostmanager-end
When I visit any of these aliases I get blank page. If I SSH into vagrant and go to var/www folder, I can see my project, but it's not visible in browser (the project is working on LAMP but I need to use vagrant since I must upgrade my system).
Can someone please help me with this, I've been fighting with this all day and I need to work on project.

**Comment by jacksonj04** [comment]
It looks like you've got your `docroot` in config.yaml set to `/var/www/awesome`, and I'm guessing you just want `/var/www`?

**Comment by alenn-m** [comment]
@jacksonj04 I just updated per your instructions but it's the same. I executed vagran provision and vagrant reload (just in case). When I run vagran ssh I can't see my project files although I can see in Terminal that /vagrant folder is synced with my project folder. If I go to /var/www I can see my project.
This is my first time using vagrant and puphpet so I really don't know what to do here.

In previous configuration (when docroot was on /var/www/awesome) I created index.php and I could see that file in browser but I didn't know how to move my project to that folder.
